client/me/purchases/cancel-private-registration/index.jsx

```diff
@@ -132,7 +127,7 @@ const CancelPrivateRegistration = React.createClass( {
 		return (
```

Comment:
On line 42 there is another `isDataLoading` method which also can be refactored.